assets/js/modules/analytics-4/components/audience-segmentation/dashboard/InfoNoticeWidget/index.test.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * InfoNoticeWidget tests.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+	render,
+	provideModules,
+	createTestRegistry,
+	provideModuleRegistrations,
+	provideUserInfo,
+	fireEvent,
+} from '../../../../../../../../tests/js/test-utils';
+import { withWidgetComponentProps } from '../../../../../../googlesitekit/widgets/util';
+import { CORE_USER } from '../../../../../../googlesitekit/datastore/user/constants';
+import InfoNoticeWidget from '.';
+import { AUDIENCE_INFO_NOTICE_SLUG, AUDIENCE_INFO_NOTICES } from './constant';
+import { WEEK_IN_SECONDS } from '../../../../../../util';
+
+describe( 'InfoNoticeWidget', () => {
+	let registry;
+
+	let dismissPromptSpy;
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+		provideModules( registry, [
+			{
+				active: true,
+				connected: true,
+				slug: 'analytics-4',
+			},
+		] );
+		provideModuleRegistrations( registry );
+		provideUserInfo( registry );
+
+		dismissPromptSpy = jest.spyOn(
+			registry.dispatch( CORE_USER ),
+			'dismissPrompt'
+		);
+	} );
+
+	const WidgetWithComponentProps =
+		withWidgetComponentProps( 'InfoNoticeWidget' )( InfoNoticeWidget );
+
+	it( 'info notice should not render when permanently dismissed', async () => {
```

Comment:
The test description should read like a natural sentence, e.g. "it should do something".

```suggestion
	it( 'should not render when permanently dismissed', async () => {
```